src/coreComponents/constitutive/docs/CO2Brine.rst

```diff
@@ -1,80 +1,256 @@
+Using the reduced volume, :math:`V_r`, we compute the fugacity coefficient of CO2, :math:`\ln_{\phi}(p,T)`, using equation (A6) of Duan and Sun (2003).
+To conclude this preprocessing step, we use the fugacity coefficient of CO2 to compute and store the solubility of CO2 in brine, :math:`s_{CO2}`, using equation (6) of Duan and Sun (2003):
 
- .. code-block:: c
+.. math::
+   \ln \frac{ x_{CO2} P }{ s_{CO2} } = \frac{\Phi_{CO2}}{RT} - \ln_{\phi}(p,T) + \sum_c 2 \lambda_c m + \sum_a 2 \lambda_a m + \sum_{a,c} \zeta_{a,c} m^2
 
-  double PVTPackage::CO2Model::computeVisc(double P, double T)
+where :math:`\Phi_{CO2}` is the chemical potential of the CO2 component, :math:`R` is the gas constant, and :math:`m` is the salinity.
+The mole fraction of CO2 in the vapor phase, :math:`x_{CO2}`, is computed with equation (4) of Duan and Sun (2003).
+Note that the first, third, fourth, and fifth terms in the equation written above are approximated using equation (7) of Duan and Sun (2003) as recommended by the authors.
 
-``input parameters:``
+During the simulation, Step 1 starts with a look-up in the precomputed table to get the CO2 solubility, :math:`s_{CO2}`, as a function of pressure and temperature.
+Then, we compute the phase fractions as:
 
-* P - pressure (Pa),  P < 300 MPa
-* T - temperature (K) 200K <= T <= 1500K 
+.. math::
+   \nu_{\ell} &= \frac{1 + s_{CO2}}{1 + z_{CO2} / ( 1 - z_{CO2} ) } \\
+   \nu_{g} &= 1 - \nu_{\ell}
 
-``return parameter:`` 
+We conclude Step 1 by computing the phase component fractions as:
 
-* viscosity (Pa.s)
+.. math::
+   y_{CO2,\ell} &= \frac{ s_{CO2} }{ 1 + s_{CO2} } \\
+   y_{H2O,\ell} &= 1 - y_{CO2,\ell} \\
+   y_{CO2,g} &= 1 \\
+   y_{H2O,g} &= 0 
+    
+   
+Step 2: Computation of the phase densities and phase viscosities
+================================================================
 
-**Function of brine density calculation**
+Gas density and viscosity
+-------------------------
 
- .. code-block:: c
+In GEOSX, the computation of the CO2 gas density and viscosity  is entirely based on look-up in precomputed tables.
+The user defines the pressure (in Pascal) and temperature (in degrees Celsius) axis of the density table in the form:
 
-  double PVTPackage::BrineModel::computeMassDensity(double P, double T, double salinity)
++------------+----------------------+-----------------+-----------------+------------------+-----------------+-----------------+------------------+
+| DensityFun | SpanWagnerCO2Density | :math:`p_{min}` | :math:`p_{max}` | :math:`\Delta p` | :math:`T_{min}` | :math:`T_{max}` | :math:`\Delta T` |
++------------+----------------------+-----------------+-----------------+------------------+-----------------+-----------------+------------------+
 
-``input parameters:``
+This correlation is valid for pressures inferior to :math:`8 \times 10^8` Pascal and temperatures inferior to 800 degrees Celsius.  
```

Comment:
```suggestion
This correlation is valid for pressures less than :math:`8 \times 10^8` Pascal and temperatures less than 800 degrees Celsius.  
```

Reply:
I fixed the grammar mistakes (this one and the ones below). Thanks for catching them